ui/src/context/ScenarioContext.tsx

```diff
@@ -19,9 +37,23 @@ export type ScenarioState = {
   select : (name : string) => Scenario
 }
 
+const structuring  = <App key={0} label="Structuring"  description="Structure and design new assets"         image={structuringImage}   path="/structuring/instruments" />;
+const issuance     = <App key={1} label="Issuance"     description="Issue new assets"                        image={issuanceImage}      path="/issuance/issuances" />;
+const custody      = <App key={2} label="Custody"      description="Manage assets in custody"                image={custodyImage}       path="/custody/assets" />;
+const distribution = <App key={3} label="Distribution" description="Distribute assets in the primary market" image={distributionImage}  path="/distribution/auctions" />;
+const lending      = <App key={4} label="Lending"      description="Borrow and lend securities"              image={lendingImage}       path="/lending/trades" />;
+const servicing    = <App key={5} label="Servicing"    description="Service and lifecycle your assets"       image={lifecyclingImage}   path="/servicing/instruments" />;
+const simulation   = <App key={6} label="Simulation"   description="Run market scenarios on your assets"     image={simulationImage}    path="/simulation/scenario" />;
+const listing      = <App key={7} label="Listing"      description="List your assets on trading venues"      image={listingImage}       path="/listing/listings" />;
+const trading      = <App key={8} label="Trading"      description="Trade assets in the secondary market"    image={tradingImage}       path="/trading/markets" />;
+const network      = <App key={9} label="Network"      description="Explore the distributed ledger network"  image={networkImage}       path="/network/overview" />;
+
 export const scenarios : Scenario[] = [
   {
-    name: "Default",
+    label: "Default",
+    description: "Primary and secondary markets workflows",
+    image: defaultImage,
+    apps: [ structuring, issuance, custody, distribution, servicing, simulation, listing, trading, network ],
     positions: new Map([
       [ "Operator",     { x:    0, y:   0 } ],
       [ "Public",       { x:    0, y:   0 } ],
```

Comment:
Hardcoded parties... Guess it's not an issue since this is a demo app only.

Reply:
Yeah, this will likely go away - it's just used to position the boxes in the "network" login screen. I've come around to the realization that the network login screen in more confusing than helping, so will revert that to a simple login form again.